.github/workflows/deploy-dev.yml

```
@@ -0,0 +1,212 @@
+    if: needs.detect-changes.outputs.has_modules == 'true'
+    concurrency:
+      group: deploy-dev-runtime-${{ github.ref }}
+      cancel-in-progress: false
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix: ${{ fromJSON(needs.detect-changes.outputs.matrix) }}
+    uses: ./.github/workflows/_ansible-exec.yml
+    with:
+      environment_name: dev
+      ssh_host: ${{ secrets.DEV_SSH_HOST }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)